src/Service/SettingsSaveService.php

```diff
@@ -213,6 +214,7 @@ public function saveSettings(&$errors = [])
             Configuration::updateValue(Config::MOLLIE_PROFILE_ID, $mollieProfileId);
```

Comment:
As you made function foreaching deletion of the configs, you can do the same here for creating them.